script.js

```diff
@@ -0,0 +1,210 @@
+const hambuger = document.querySelector(' .hambuger');
+hambuger.addEventListener('click', () => {
+  hambuger.classList.toggle('active');
+});
+const anchor = document.querySelectorAll('.nav-item');
+for (let i = 0; i < anchor.length; i += 1) {
+  anchor[i].addEventListener('click', () => {
+    hambuger.classList.remove('active');
+  });
+}
+
+/* Javscript code for Popup window */
+
+const project = [
+  {
+    name: 'Tonic',
+    li1: 'Canopy',
+    li2: 'Back End dev',
+    li3: '2015',
+    project_img: 'images/Snapshoot Portfolio.png',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s",
+    li4: 'Html',
+    li5: 'Css',
+    li6: 'Javascript',
+    live_button: 'See live',
+    online_button: 'See source',
+  },
+  {
+    name: 'Tonic',
+    li1: 'Canopy',
+    li2: 'Back End dev',
+    li3: '2015',
+    project_img: 'images/Snapshoot Portfolio-2.png',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s",
+    li4: 'Html',
+    li5: 'Css',
+    li6: 'Javascript',
+    live_button: 'See live',
+    online_button: 'See source',
+  },
+  {
+    name: 'Tonic',
+    li1: 'Canopy',
+    li2: 'Back End dev',
+    li3: '2015',
+    project_img: 'images/Snapshoot Portfolio-3.png',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s",
+    li4: 'Html',
+    li5: 'Css',
+    li6: 'Javascript',
+    live_button: 'See live',
+    online_button: 'See source',
+  },
+  {
+    name: 'Tonic',
+    li1: 'Canopy',
+    li2: 'Back End dev',
+    li3: '2015',
+    project_img: 'images/Snapshoot Portfolio-4.png',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s",
+    li4: 'Html',
+    li5: 'Css',
+    li6: 'Javascript',
+    live_button: 'See live',
+    online_button: 'See source',
+  },
+];
```

Comment:
The JavaScript code implements an array of objects to store JavaScript objects containing the information for each project 🙌 Well done!

Reply:
Hi  Mateo Villagomez good to hear from you, 
I don't understand how I can  set up Eslint for JS because I have done that already and also I checked for the linters error.

Reply:
As I mentioned, your [.github](https://github.com/ifeanyi305/portfolio/tree/Popup-window/.github)/[workflows](https://github.com/ifeanyi305/portfolio/tree/Popup-window/.github/workflows)/linters.yml is causing the issue. The file does not contain eslint triggering info. Please be sure to change your file contents to this [one](https://github.com/microverseinc/linters-config/blob/master/html-css-js/.github/workflows/linters.yml).